content/en/docs/zero-code/java/quarkus.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Quarkus instrumentation
+linkTitle: Quarkus
+---
+
+## Quarkus OpenTelemetry instrumentation
+
+[Quarkus](https://quarkus.io/) is an open source framework designed to help
+software developers build efficient cloud native applications both with JVM and
+Quarkus native image applications.
+
+Quarkus uses extensions to provide optimized support for a wide range of
+libraries. The
+[Quarkus OpenTelemetry extension](https://quarkus.io/guides/opentelemetry)
+provides:
+
+- Out of the box instrumentation
+- OpenTelemetry SDK autoconfiguration, supporting almost all system properties
+  defined for the [OpenTelemetry SDK](/docs/languages/java/configuration/)
+- [Vert.x](https://vertx.io/) based OTLP exporter
+- The same instrumentations can be used with native image applications, which
+  are not supported by the OpenTelemetry Java agent.
+
+> [!NOTE] Quarkus OpenTelemetry instrumentation is maintained and supported by
+> Quarkus. For details, please check the
+> [Quarkus community support page](https://quarkus.io/support/).
```

Comment:
The website does not yet support GH-style alerts as you can see:

> <img width="555" alt="image" src="https://github.com/user-attachments/assets/622b673b-0c4f-41c0-ade5-1b9cd438d0f9" />

Let's use an alert here:

```suggestion
{{% alert-md title="Note" color="secondary" %}}

Quarkus OpenTelemetry instrumentation is maintained and supported by Quarkus.
For details, see [Quarkus community support](https://quarkus.io/support/).

{{% /alert-md %}}
```